UPGRADE-2.0.md

```diff
@@ -294,7 +331,7 @@
 
 - The `Raven_Client::getLastEventID` method has been removed. The ID of the
   last event that was captured is now returned by each of the `Client::capture*`
-  methods.
+  methods. Also you can use `Hub::getCurrent()->getLastEventId()`.
```

Comment:
I believe that saying `You can also use [...]` is  better from a grammatical point of view